src/index.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+import SQLModule from './sql/sql';
```

Comment:
this path could be redundant and confuse, what you think about of rename the sql file for another name ?